web/templates/popovers/message_actions_popover.hbs

```diff
@@ -1,132 +1,120 @@
+        <li role="separator" class="popover-menu-separator"></li>
+        {{/if}}
+        {{#if should_display_mark_as_unread}}
+        <li role="none" class="link-item popover-menu-list-item">
+            <a role="menuitem" data-message-id="{{message_id}}" class="mark_as_unread popover-menu-link" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-mark-as-unread" aria-hidden="true"></i>
+                <span class="popover-menu-label">{{t "Mark as unread from here" }}</span>
+                {{popover_hotkey_hints "Shift" "U"}}
+            </a>
+        </li>
+        {{/if}}
+        {{#if should_display_hide_option}}
+        <li role="none" class="link-item popover-menu-list-item">
+            <a role="menuitem" data-message-id="{{message_id}}" class="rehide_muted_user_message popover-menu-link" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-hide" aria-hidden="true"></i>
+                <span class="popover-menu-label">{{t "Hide muted message again" }}</span>
+            </a>
+        </li>
+        {{/if}}
+        {{#if should_display_collapse}}
+        <li role="none" class="link-item popover-menu-list-item">
+            <a role="menuitem" data-message-id="{{message_id}}" class="popover_toggle_collapse popover-menu-link" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-collapse" aria-hidden="true"></i>
+                <span class="popover-menu-label">{{t "Collapse message" }}</span>
+                {{popover_hotkey_hints "-"}}
+            </a>
+        </li>
+        {{/if}}
+        {{#if should_display_uncollapse}}
+        <li role="none" class="link-item popover-menu-list-item">
+            <a role="menuitem" data-message-id="{{message_id}}" class="popover_toggle_collapse popover-menu-link" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-expand" aria-hidden="true"></i>
+                <span class="popover-menu-label">{{t "Expand message" }}</span>
+                {{popover_hotkey_hints "-"}}
+            </a>
+        </li>
+        {{/if}}
+        {{!-- Group 5 --}}
+        <li role="separator" class="popover-menu-separator"></li>
+        {{#if view_source_menu_item}}
+        <li role="none" class="link-item popover-menu-list-item">
+            <a role="menuitem" data-message-id="{{message_id}}" class="popover_view_source popover-menu-link" tabindex="0">
+                <i class="popover-menu-icon zulip-icon zulip-icon-source" aria-hidden="true"></i>
+                <span class="popover-menu-label">{{view_source_menu_item}}</span>
+                {{popover_hotkey_hints "E"}}
+            </a>
         </li>
         {{/if}}
-        {{#if (or should_display_mark_as_unread should_display_reminder_option should_display_hide_option should_display_collapse should_display_uncollapse)}}
```

Comment:
It looks like you removed `should_display_reminder_option`, which is correct as it's unused, but makes the diff harder to read, so definitely should have been in its own commit.

Reply:
My bad, thanks for pointing that out!